pkg/ovs/ovsctl/ofctl.go

```diff
@@ -63,6 +64,27 @@ func (c *ovsCtlClient) DumpTableFlows(table uint8) ([]string, error) {
 	return c.DumpFlows(fmt.Sprintf("table=%d", table))
 }
 
+func (c *ovsCtlClient) DumpGroup(groupID int) (string, error) {
+	// There seems a bug in ovs-ofctl that dump-groups always returns all
```

Comment:
out-of-curiosity did you report the bug?

Reply:
Not yet. Will do.